logprep/run_logprep.py

```diff
@@ -141,6 +141,8 @@ def main():
     try:
         AggregatingLogger.setup(config, logger_disabled=args.disable_logging)
         logger = AggregatingLogger.create("Logprep")
+        logger.info(f"python version: {sys.version}")
+        logger.info(f"logprep version: {get_versions()}")
```

Comment:
If you are at it already, how about we log also the found configuration version?

Reply:
done